### PR TITLE
First cleanup of Javadoc comments

### LINKED ISSUE #672: Update Javadoc comments
according to #642

* remove `Changelog:`-Section
* remove `Issues:`- and `Todos:`-Section
* remove `Description:`-Prefix

---

**EDIT by SecUpwN.** I have crawled through our *whole* code and below files need *heavy* cleanup.
Please add a checkmark when a file is cleaned up so that we know what still needs to be worked on:

- [ ] AIMSICDDbAdapter.java
- [ ] BtsMeasureCardInflater.java
- [ ] BtsMeasureItemData.java
- [ ] CardItemData.java
- [x] CellCardInflater.java
- [ ] EventLogItemData.java
- [ ] UniqueBtsCardInflater.java
- [ ] UniqueBtsItemData.java
- [ ] DebugLogs.java
- [ ] MapViewerOsmDroid.java
- [ ] AtCommandFragment.java
- [ ] CellInfoFragment.java
- [ ] DbViewerFragment.java
- [ ] AccelerometerMonitor.java
- [ ] AimsicdService.java
- [ ] CellTracker.java
- [ ] SignalStrengthTracker.java
- [ ] SmsDetector.java
- [ ] AtCommandTerminal.java
- [ ] TtyStream.java
- [x] Device.java
- [ ] Helpers.java
- [ ] RequestTask.java
- [ ] StackOverflowXmlParser.java
- [x] SystemPropertiesReflection.java
- [x] AIMSICD.java

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/28951923-update-javadoc-comments?utm_campaign=plugin&utm_content=tracker%2F565750&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F565750&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by SecUpwN**
This PR is the start of multiple PRs to clean up our codebase from the following sections to fix #672:

* Comments and whole conversations
* Issues (moved into GitHub Issues)
* TODO (moved into GitHub Issues)

~~I have kept the `Description` prefix where applicable, in some files a small description is better.~~
@larsgrefer, a list of still dirty files is in the OP of #672, maybe you can add fixes for those here?

### CODE REVIEW

File: AIMSICD/src/main/java/com/SecUpwN/AIMSICD/activities/OpenCellIdActivity.java [hunk 1]
```diff
@@ -28,9 +28,6 @@
 /**
  *  Description:    Popup toast messages asking if user wants to download new API key
```
Comment (larsgrefer) [inline AIMSICD/src/main/java/com/SecUpwN/AIMSICD/activities/OpenCellIdActivity.java]: Please remove the `Description:` here to make this a proper javadoc-comment

File: AIMSICD/src/main/java/com/SecUpwN/AIMSICD/adapters/EventLogCardInflater.java [hunk 2]
```diff
@@ -13,19 +13,12 @@
...
- *      This class handle the EventLog DB table ... TODO: Add info here !
- *
- *  Template:       OpenCellIdCardInflater.java
- *
- *  Dependencies:   EventLogItemData.java
- *                  eventlog_items.xml
- *
- *  Issues:
- *  Notes:
+ *  Dependencies: EventLogItemData.java
```
Comment (larsgrefer) [inline AIMSICD/src/main/java/com/SecUpwN/AIMSICD/adapters/EventLogCardInflater.java]: Why do we need the `Dependencies:` block here?

Comment (SecUpwN) [inline AIMSICD/src/main/java/com/SecUpwN/AIMSICD/adapters/EventLogCardInflater.java]: >Why do we need the `Dependencies:` block here?

@larsgrefer, the `Dependencies:` block is contained in many files within our code, yet this one looks self-explanatory. Shall I remove it?

Comment (larsgrefer) [inline AIMSICD/src/main/java/com/SecUpwN/AIMSICD/adapters/EventLogCardInflater.java]: I think we have the `Find usages` feature in Android Studio to find dependencies so we ca remove it here

Comment (SecUpwN) [inline AIMSICD/src/main/java/com/SecUpwN/AIMSICD/adapters/EventLogCardInflater.java]: Ok, fair. Shall I remove all `Dependencies:` sections in all files that contain them?

**Comment by SecUpwN** [comment]
@larsgrefer, I likely removed an exit function or something. Would you please help me fix it?

**Comment by larsgrefer** [comment]
@SecUpwN Just look at the travis logs: https://travis-ci.org/SecUpwN/Android-IMSI-Catcher-Detector/builds/99470524#L1799-L1800

**Comment by SecUpwN** [comment]
@larsgrefer, this first part of the cleaning is done. Files that need harder cleaning reside in the OP of #672. Please verify and directly merge this PR, I'll be going to bed soon since I've only slept 4 hours.

**Comment by larsgrefer** [comment]
Approved

**Pull request merged**